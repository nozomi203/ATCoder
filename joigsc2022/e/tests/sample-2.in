6 160
63
63
63
63
63
63
