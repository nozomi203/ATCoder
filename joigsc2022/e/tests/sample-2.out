3 126
3 63
2 126
2 63
1 126
1 63
